MSCL/source/mscl/MicroStrain/Inertial/ExposedInertialTypes.h

```diff
@@ -2971,6 +2972,89 @@ namespace mscl
         EventTriggerParameters parameters;
     };
 
+    //API Struct: EventActionGpioParameters
+    struct EventActionGpioParameters
+    {
+        //API Enum: Mode
+        //  Modes for behavior of the GPIO pin
```

Comment:
Add a space after the description. NaturalDocs won't generate these types of documentation properly without it. Take a look at line 2952 EventTriggerConfiguration::Type for what I'm talking about